addon/src/components/animated-if.ts

```diff
@@ -39,7 +41,7 @@ import { computed } from '@ember/object';
         sprite.endAtPixel({ x: window.innerWidth });
         move(sprite, { easing: easeIn });
       }
-    },
+    }
   });
```

Comment:
now we should remove extra closing `)`

```suggestion
  };
```